kore/src/Kore/BugReport.hs

```diff
@@ -109,6 +110,8 @@ withBugReport exeName bugReport act =
         case exitCase of
             ExitCaseSuccess _ -> optionalWriteBugReport tmpDir
             ExitCaseException someException
+              | Just ExitSuccess == fromException someException ->
```

Comment:
It's possible the constructor names are confusing here, or I'm missing some information, but how can a crash triggered by an exception be a successful run?

Reply:
When a command returns the status `SuccessStop` we call `exitSuccess` which is equivalent to `exitWith ExitSuccess`. My understanding of [`exitWith`](http://localhost:8000/file/home/andrei/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/share/doc/ghc-8.6.5/html/libraries/base-4.12.0.0/System-Exit.html#v:exitWith) is that an exception is thrown.

Reply:
Ah, I see. Since this is `repl` specific, could you add a comment here describing that this corresponds to the user exiting the `repl` after the proof was finished?